doc/integration.rst

```diff
@@ -6,18 +6,18 @@ Application Integration and Configuration
 Overview
 ========
 
-Softbank specifies that your certified product must be able to update its modem firmware, regardless other tasks and operations your main application is running.
-The Softbank FOTA library is an independent library that is intended to runs together with your application when included in your project. See the below image.
+SoftBank specifies that your certified product must be able to update its modem firmware, regardless other tasks and operations your main application is running.
+The SoftBank FOTA library is an independent library that is intended to run together with your application when included in your project. See the below image.
 
 .. figure:: images/softbank_fota_summary.svg
-   :alt: Interaction between the Softbank FOTA library and the user application
+   :alt: Interaction between the SoftBank FOTA library and the user application
 
-   Interaction between the Softbank FOTA library and the user application
+   Interaction between the SoftBank FOTA library and the user application
 
 OS layer
 ========
 
-The Softbank FOTA library has an OS abstraction layer.
+The SoftBank FOTA library has an OS abstraction layer.
 This abstraction layer makes the softbank library independent of some |NCS| modules and underlying implementation of primitives such as timers, non-volatile storage, and heap allocation.
 For more information, see the :file:`sb_fota_os.h` file or the :ref:`sb_fota_os` section for available APIs. It provides an abstraction of the following modules:
```

Comment:
Can't comment on those lines, but we should update the list below, I think none of the NCS modules are right...

Reply:
Updated the list. It should be correct for the current bitbucket revision.